samples/snippets/src/test/java/aiplatform/UpdateFeaturestoreFixedNodesSampleTest.java

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
Similar to other PRs, let's bundle several tests in one test case to reduce the overhead cost for creates a Feature Store. My suggestion is:
- One test that does Create, Get/List, Update(s) & Delete Featurestore (tear down)
- One test that does Create, Get/List Update & Delete EntityType. Create & Delete FS can be startup and tear down for the test.
- One test that does Create, Get/List, Update & Delete Feature.